docs/pages/deploy-a-cluster/helm-deployments/kubernetes-cluster.mdx

```diff
@@ -165,7 +166,15 @@ to create a public IP for Teleport.
   </TabItem>
 </Tabs>
 
+{/* Disabling the DNS partial until it's updated to cover this scope.
 (!docs/pages/includes/dns.mdx!)
+*/}
+
+Configure DNS entries for `tele.example.com` for traffic to the Teleport Proxy
+web UI, and `*.tele.example.com` for web apps using Application Access, using `A`
+records for an IP address and `CNAME` records for a hostname. The record
+type and configuration will depend on your Kubernetes provider, see their
```

Comment:
```suggestion
Configure DNS entries for `tele.example.com` for traffic to the Teleport Web UI, and `*.tele.example.com` for web apps using Application Access, using `A`
records for an IP address and `CNAME` records for a hostname. The record
type and configuration will depend on your Kubernetes provider, see their
```

Reply:
Also, the instructions in the `Tabs` before this show you how to tell whether the load balancer has an existing A record or not (i.e., if `SERVICE_IP` points to an IP address or hostname). Should we tell the user to create a `CNAME` record if `SERVICE_IP` is a hostname and an `A` record if it's an IP address, rather than asking them to search external docs?